db-engine/src/main/java/com/splicemachine/db/iapi/sql/dictionary/DDUtils.java

```diff
@@ -261,7 +261,7 @@ public static void validateReferentialActions(DataDictionary dd,
                                                               deleteConnectionsMap,
                                                               newForeignKeyConstraintName);
             }
-        }
+        } // would be nice to have a logger so we can log that we are not going to perform foreign key checks.
```

Comment:
you can define `    private static final Logger LOG = Logger.getLogger(DDUtils.class);` at the top of the class.